content/docs/gherkin/reference.md

```diff
@@ -303,6 +305,35 @@ Feature: Multiple site support
     Then I should see "Your article was published."
 ```
 
+`Background` is also supported at the `Rule` level, for example:
+
+```gherkin
+Feature: Overdue tasks
+  Let users know when tasks are overdue, even when using other
+  features of the app
+
+  Rule: Users are notified about overdue tasks on first use of the day
+    Background:
+      Given I have overdue tasks
+
+    Example: First use of the day
+      Given I last used the app yesterday
+      When I use the app
+      Then I am notified about overdue tasks
+
+    Example: Already used today
+      Given I last used the app earlier today
+      When I use the app
+      Then I am not notified about overdue tasks
+  ...
+```
+
+{{% warn "Use with caution"%}}
+Whilst usage of `Background` within `Rule` is currently supported, it's not recommended, and may be removed in future versions of Gherkin.
```

Comment:
Is it supported in all implementations (or JavaScript only)? Maybe point to the issue for removal (even if stale)?

Reply:
(Suggestion to make this extra clear; not blocking for merge imho)

Reply:
Hmm good point. On the face of it I would say yes, any implementations support `Rule` would currently work with `Background` under it, since it would be a by-product of the compilation to pickles that happens in Gherkin, which should be consistent across implementations.

There are like six things I could have got wrong in that sentence though - maybe one to check with the maintainers for the other languages?